gradio/routes.py

```diff
@@ -644,9 +650,17 @@ async def sse_stream(request: fastapi.Request):
                                     )
                                 ):
                                     return
-                except asyncio.CancelledError as e:
-                    del blocks._queue.pending_messages_per_session[session_hash]
-                    await blocks._queue.clean_events(session_hash=session_hash)
+                except Exception as e:
```

Comment:
Has to be BaseException to catch everything - asyncio.CancelledError is not an instance of Exception